site/model/stores.js

```diff
@@ -40,6 +40,13 @@ exports.stores = {
         getUrl: (item) => `https://www.penny.cz/products/${item.url}`,
         removeOld: true,
     },
+    dm: {
+        name: "DM",
+        budgetBrands: [],
+        color: undefined,
```

Comment:
budget brand a color by mohlo byt okopirovano z puvodni rakouske verze, pridamo to v naslednem commitu